src/hooks/waste.tsx

```diff
@@ -287,44 +287,54 @@ export const useRenderSuggestions = (selectionCallback?: () => void) => {
 };
 
 export const useTriggerExport = ({ streetData, wasteTexts }) => {
-  const triggerExport = useCallback(() => {
-    const { street, zip, city } = getLocationData(streetData);
+  const triggerExport = useCallback(
+    ({ selectedTypes }) => {
+      const { street, zip, city } = getLocationData(streetData);
 
-    const baseUrl = secrets[namespace].serverUrl + staticRestSuffix.wasteCalendarExport;
+      const baseUrl = secrets[namespace].serverUrl + staticRestSuffix.wasteCalendarExport;
 
-    let params = `street=${encodeURIComponent(street)}`;
+      let params = `street=${encodeURIComponent(street)}`;
 
-    if (zip) {
-      params += `&zip=${encodeURIComponent(zip)}`;
-    }
+      if (!!zip) {
+        params += `&zip=${encodeURIComponent(zip)}`;
+      }
 
-    if (city) {
-      params += `&city=${encodeURIComponent(city)}`;
-    }
+      if (!!city) {
+        params += `&city=${encodeURIComponent(city)}`;
+      }
 
-    const combinedUrl = baseUrl + params;
+      if (!!selectedTypes) {
+        params += `&waste_types=${Object.keys(selectedTypes)
+          .filter((key) => selectedTypes[key])
+          .map((key) => key.replace(/_/g, ''))
```

Comment:
what is `key.replace(/_/g, ''))` needed for?

Reply:
This was an unnecessary code. It is now removed and we don't use it anymore.